arrow-libs/core/arrow-autocloseable/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+@file:Suppress("DSL_SCOPE_VIOLATION")
+
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+
+plugins {
+  id(libs.plugins.kotlin.multiplatform.get().pluginId)
+  alias(libs.plugins.arrowGradleConfig.kotlin)
+  alias(libs.plugins.arrowGradleConfig.publish)
+  alias(libs.plugins.spotless)
+}
+
+spotless {
+  kotlin {
+    ktlint().editorConfigOverride(mapOf("ktlint_standard_filename" to "disabled"))
+  }
+}
+
+apply(from = property("ANIMALSNIFFER_MPP"))
+
+kotlin {
+  sourceSets {
+    commonMain {
+      dependencies {
+        implementation(libs.kotlin.stdlibCommon)
```

Comment:
nit: Isn't stdlib brought in by default

Reply:
I don't think so, I also had to add `jvmMain` and `jsMain` to depend on `jvm` and `js` stdlib. I think there is still something broken in our Gradle setup..

In general I think it's not needed, but seems to be needed in our setup.

Reply:
This weirdness is fixed in the 2.0 branch, there you only need to depend on the common stdlib.